js/index.js

```diff
@@ -3,40 +3,100 @@
 function updateSubtotal(product) {
   console.log('Calculating subtotal, yey!');
 
-  //... your code goes here
+  const price = product.querySelector('.price span').innerHTML;
```

Comment:
```suggestion
  const price = product.querySelector('.price span').innerText;
```
Siempre que puedas usar el innerText mejor que el innerHTML, el HTML para casos especiales y cuando no haya mas opcion, si no caca XD